src/SB/Core/x/xMath.cpp

```diff
@@ -436,3 +436,13 @@ void xAccelStop(F32& x, F32& v, F32 a, F32 dt) {
         v = 0.0f;
     }
 }
+
+void xFuncPiece_EndPoints(xFuncPiece* func, F32 pi, F32 pf, F32 fi, F32 ff) {
+    float xfinv;
```

Comment:
Is this from the DWARF data? If so, you can leave it in, but if not, please remove it. 

If you leave it, please change its type to F32.